firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/crcelements.txt

```diff
@@ -0,0 +1,93 @@
+/* compute_body_crc
```

Comment:
The way CRCElementReceiver and BackendReceiver use ParsedCRCElement is they construct a ParsedCRCElement with a buffer to store the CRCElement's payload as the parse output. The ParsedCRCElement's constructor takes a non-const buffer, which means that it reserves the right to modify the buffer given through the constructor. So the comment at https://github.com/pez-globo/pufferfish-software/blob/eb742d07167caf2d1700923b2b94f5fedc3f71d2/firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Protocols/CRCElements.h#L61 should be swapped with the comment at https://github.com/pez-globo/pufferfish-software/blob/eb742d07167caf2d1700923b2b94f5fedc3f71d2/firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Protocols/CRCElements.h#L67 - let's include this in #281 .